aptos-move/framework/aptos-framework/sources/reconfiguration.spec.move

```diff
@@ -58,33 +72,49 @@ spec aptos_framework::reconfiguration {
         aborts_if !exists<Configuration>(@aptos_framework);
         let config_ref = global<Configuration>(@aptos_framework);
         aborts_if !(config_ref.epoch == 0 && config_ref.last_reconfiguration_time == 0);
+        ensures global<Configuration>(@aptos_framework).epoch == 1;
     }
 
     spec last_reconfiguration_time {
         aborts_if !exists<Configuration>(@aptos_framework);
+        ensures result == global<Configuration>(@aptos_framework).last_reconfiguration_time;
     }
 
     spec reconfigure {
-        use aptos_framework::aptos_coin;
+        use aptos_framework::coin::CoinInfo;
+        use aptos_framework::aptos_coin::AptosCoin;
         use aptos_framework::transaction_fee;
         use aptos_framework::staking_config;
 
         pragma verify_duration_estimate = 120; // TODO: set because of timeout (property proved)
 
         requires exists<stake::ValidatorFees>(@aptos_framework);
+        requires exists<CoinInfo<AptosCoin>>(@aptos_framework);
 
         include transaction_fee::RequiresCollectedFeesPerValueLeqBlockAptosSupply;
-        include features::spec_periodical_reward_rate_decrease_enabled() ==> staking_config::StakingRewardsConfigEnabledRequirement;
-        include features::spec_collect_and_distribute_gas_fees_enabled() ==> aptos_coin::ExistsAptosCoin;
-
+        include staking_config::StakingRewardsConfigRequirement;
```

Comment:
why do you change this?

Reply:
It should be a error in conflict solve, it has been added back

Reply:
It has not been added back from what I see here

Reply:
`include staking_config::StakingRewardsConfigRequirement;` is not necessary. Please remove it

Reply:
removed